src/MonitoringDemo/Program.cs

```diff
@@ -77,7 +77,7 @@ static async Task Main(string[] args)
 
         private static void ScaleSalesEndpointIfRequired(DemoLauncher launcher, TaskCompletionSource<bool> syncEvent)
         {
-            Task.Run(() =>
+            _ = Task.Run(() =>

```

Comment:
Given line https://github.com/Particular/MonitoringDemo/pull/77/files#diff-49b83d8b1f1710f14f7bd20bf256cdddR111, I assume dropping this task is deliberate.